css/quiz.css

```diff
@@ -0,0 +1,84 @@
+* {
```

Comment:
Your indentation has mixed tabs and spaces.  While the holy war on indentation can go either way for the tabs vs. spaces camp, the indentation should be consistent throughout the document/project.  Pick the indentation style you'd prefer and apply it throughout the document.